partiql-planner/src/test/kotlin/org/partiql/planner/internal/typer/PlanTyperTestsPorted.kt

```diff
@@ -2144,15 +2136,13 @@ class PlanTyperTestsPorted {
                     ) FROM <<
                         { 'a': { 'b': 1 } },
                         { 'a': { 'b': 'hello' } },
-                        { 'a': NULL },
```

Comment:
I updated this only because the literal NULL's type is unknown. Which leads to the inferred type to be ANY when used as an argument to a literal struct.

Reply:
Not too sure why the empty struct in the input binding tuples doesn't cause `ANY` for the type. Perhaps that binding tuple is filtered out?

Reply:
Not sure if there's a test showing this but if there's not, could still be helpful to include a test showing that the inferred type for a path expression on a literal struct with a `NULL`/`MISSING` value is `ANY`. E.g. `{ 'a': NULL }.a` -> `ANY` and `{ 'a': MISSING }.a` -> `ANY`